### PR TITLE
Allow "from klein import handle_errors"

### PR DESCRIPTION
**Pull request by IlyaSkriblovsky**
Makes it possible to `from klein import handle_errors` just like `route`, `run` and `resource`. Useful for writing quick tests and examples.

### CODE REVIEW

**Comment by IlyaSkriblovsky** [comment]
Here you are :)
Btw, other methods of global app objects wasn't tested too, so this testcase fires them too

**Comment by wsanchez** [comment]
Actually, can we pause here?

I'd like to limit the new API exposure (even if it's an existing symbol just in a new place), because we've been discussing cleaning up the API to be consistent with Twisted (and itself), so I'm a bit wary of adding more symbols into the top-level module in the meantime.

Using this as an example, this should be called `handleErrors` in the long run.  Keeping this in the `klein.app` submodule will made it easier to eventually deprecate the old endpoint.

**Comment by wsanchez** [comment]
@glyph ^^

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,8 @@ docs/_build/
 .DS_Store
 dist/
 *~
+/venv3/
```
Comment (glyph) [inline .gitignore]: These `.gitignore` changes have nothing to do with this change and should be removed.

**Comment by glyph** [comment]
> I'd like to limit the new API exposure (even if it's an existing symbol just in a new place), because we've been discussing cleaning up the API to be consistent with Twisted (and itself), so I'm a bit wary of adding more symbols into the top-level module in the meantime.

I can definitely see this point (and it should definitely be renamed to `handleErrors` soon), but I also think that making the global `klein` as consistent as possible with `Klein` is generally a good thing; there's no way to access this now.  The tests that this adds will make it easier to expose `handleErrors`, too.

However, I can't merge it now due to merge conflicts.

File: src/klein/test/test_resource.py [hunk 2]
```diff
@@ -1214,6 +1214,33 @@ def test_afUnixSocket(self):
...
 
+class GlobalAppTests(TestCase):
+    """
+    Tests for the global app object
+    """
+
+    def test_global_app(self):
+        from klein import run, route, resource, handle_errors
+
+        global_app = run.__self__
```
Comment (glyph) [inline src/klein/test/test_resource.py]: also, this should be `globalApp` :)

File: .gitignore (hunk 1 above)
Comment (IlyaSkriblovsky) [inline .gitignore]: yeah, sorry

File: src/klein/test/test_resource.py (hunk 2 above)
Comment (IlyaSkriblovsky) [inline src/klein/test/test_resource.py]: Fixed. Just thought it is ok for local variable to be underscore_separated :)

**Comment by IlyaSkriblovsky** [comment]
Rebased to master and fixed some style issues

**Comment by wsanchez** [comment]
Review appears to be addressed, build OK; merging.

**Pull request merged**